modules/log6x-6-1-1-rn.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * observability/logging/logging-6.1/log6x-release-notes-6.1.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="logging-release-notes-6-1-1_{context}"]
+= Logging 6.1.1 Release Notes
+
+This release includes link:https://access.redhat.com/errata/RHBA-2024:10992[{logging-uc} {for} Bug Fix Release 6.1.1].
+
+[id="logging-release-notes-6-1-1-enhancements_{context}"]
+== New Features and Enhancements
+
+* With this update, the {loki-op} supports configuring the workload identity federation on the {gcp-first} by using the Cluster Credential Operator (CCO) in {product-title} 4.17 or later. (link:https://issues.redhat.com/browse/LOG-6420[LOG-6420])
+
+[id="logging-release-notes-6-1-1-bug-fixes_{context}"]
+== Bug Fixes
+
+* Before this update, when any infrastructure namespaces were specified in the application inputs in the `ClusterLogForwarder` custom resource, logs were generated with the incorrect `log_type: application` tags. With this update, when any infrastructure namespaces are specified in the application inputs, logs are generated with the correct `log_type: infrastructure` tags. (link:https://issues.redhat.com/browse/LOG-6373[LOG-6373])
+
+* Before this update, the collector was discarding longer audit log messages with the following error message: *Internal log [Found line that exceeds max_line_bytes; discarding.]*. With this update, the discarding of longer audit messages is avoided by increasing the audit configuration thresholds: The maximum line size, `max_line_bytes`, is `3145728` bytes. The maximum number of bytes read during a read cycle, `max_read_bytes`, is `262144` bytes. (link:https://issues.redhat.com/browse/LOG-6379[LOG-6379])
```

Comment:
It's probably correct according to some style guides but I feel the capitalization after the colon is a little weird in this context. Up to you, but I'd consider just replacing the colon with a period:
```suggestion
* Before this update, the collector was discarding longer audit log messages with the following error message: *Internal log [Found line that exceeds max_line_bytes; discarding.]*. With this update, the discarding of longer audit messages is avoided by increasing the audit configuration thresholds. The maximum line size, `max_line_bytes`, is `3145728` bytes. The maximum number of bytes read during a read cycle, `max_read_bytes`, is `262144` bytes. (link:https://issues.redhat.com/browse/LOG-6379[LOG-6379])
```